### PR TITLE
#1908: Update print plugins and configurations

### PR DESCRIPTION
**Pull request by dsuren1**
### Description
This PR adds new print plugin components (Author and Copyright) and update configs

### Issue
- #1908 

> [!NOTE]
Related config.yaml PR is here https://github.com/GeoNode/geoserver-geonode-ext/pull/191

### CODE REVIEW

File: geonode_mapstore_client/client/js/plugins/Print/Author.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+};
+
+const Author = (props) => {
+    useEffect(() => {
+        addTransformer("author", authorTransformer, 6);
+    }, []);
+    return <TextInput {...props} />;
+};
+
+export default createPlugin("PrintAuthor", {
```
Comment (allyoucanmap) [inline geonode_mapstore_client/client/js/plugins/Print/Author.jsx:33]: Please try to see if both plugin could be replace by just localConfig configuration see PrintInputText https://github.com/geosolutions-it/MapStore2/blob/master/web/client/plugins/print/TextInput.jsx#L55-L67

**Review by allyoucanmap — CHANGES_REQUESTED** [review]

File: geonode_mapstore_client/client/themes/geonode/less/ms-theme.less [hunk 2]
```diff
@@ -394,3 +394,14 @@ div#mapstore-globalspinner {
 .glyphicon {
     top: 0;
 }
+
+.gn-viewer-layout-body {
+    #mapstore-print-panel {
+        &.modal-dialog {
```
Comment (allyoucanmap) [inline geonode_mapstore_client/client/themes/geonode/less/ms-theme.less:400]: please add margin 0 to #mapstore-print-panel

**Review by allyoucanmap — CHANGES_REQUESTED** [review]
We should add these formats:
image

File: geonode_mapstore_client/client/js/plugins/Print/Author.jsx (hunk 1 above)
Comment (dsuren1) [reply geonode_mapstore_client/client/js/plugins/Print/Author.jsx:33]: Issue reference: https://github.com/geosolutions-it/MapStore2/issues/10728

**Review by allyoucanmap — APPROVED** [review]

**Pull request merged**